src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/components/SampleFiltering/index.tsx

```diff
@@ -68,25 +73,37 @@ export function SampleFiltering({
   );
   return (
     <StyledContainer>
-      <StyledExplainerTitle>
-        {isTreeLocationFilterFlagOn
-          ? "Define samples of interest by:"
-          : "Limit samples from my jurisdiction to:"}
-        {isTreeLocationFilterFlagOn ? (
-          <SampleFilteringTooltip />
-        ) : (
-          <StyledTooltip
-            arrow
-            leaveDelay={1000}
-            title={SAMPLE_FILTERING_TOOLTIP_TEXT}
-            placement="top"
+      <StyledTitleContainer>
+        <StyledExplainerTitle>
+          {isTreeLocationFilterFlagOn
```

Comment:
Instead of having 3 conditionals in a row that checks for `isTreeLocationFilterOn`, can we condense it into one? I think that might make it much easier to see what is displayed in each case